server/routes/api.js

```diff
@@ -0,0 +1,102 @@
+const express = require('express')
+const router = express.Router()
+const moment = require('moment')
+const Expense = require('../model/Expense')
+router.use(express.json())
+
+router.get('/:expenses', function (req, res) {
```

Comment:
the name of this route should be `/expenses` and not `/:expenses` - same in the POST route